sdks/python/apache_beam/ml/inference/pytorch_inference.py

```diff
@@ -400,11 +430,30 @@ def __init__(
       logging.info("Device is set to CPU")
       self._device = torch.device('cpu')
     self._model_class = model_class
-    self._model_params = model_params
+    self._model_params = model_params if model_params else {}
     self._inference_fn = inference_fn
-    self._use_torch_script_format = use_torch_script_format
+    self._batching_kwargs = {}
+    if min_batch_size is not None:
+      self._batching_kwargs['min_batch_size'] = min_batch_size
+    if max_batch_size is not None:
+      self._batching_kwargs['max_batch_size'] = max_batch_size
+    self._torch_script_model_path = torch_script_model_path
+
+    self.validate_constructor_args()
+
+  def validate_constructor_args(self):
```

Comment:
Can we share this function as `_validate_constructor_args` (like `_load_model`)

Reply:
If you're planning on doing something like that in your follow up refactor pr that you mentioned that is fine as well.

Reply:
Yes, I will take care of refactoring in the next PR.